samples/ipc_call_interface/producers/cipher_v1i1/cipher_v1i1.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#ifndef CIPHER_V1I1_H
+#define CIPHER_V1I1_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+  void cipher_v1i1_create(void);
+  void cipher_v1i1_destroy(void);
+
+  az_result cipher_v1i1_encrypt(
+      uint32_t context,
```

Comment:
For now, it was made generic of purpose because it can mean any difference between "contexts". In the future, we are thinking about a real case where we will replace this "context" by something meaningful. 

---
In reply to: [582408174](https://github.com/Azure/azure-ulib-c/pull/60#discussion_r582408174) [](ancestors = 582408174)